application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
+
+        if (list.size() > 200) {
```

Comment:
`200` is a magic value and should be replaced by a proper constant with a decent name instead. for example `MAX_TAGS_WARNING_THRESHOLD`

Reply:
> Could you share a screenshot of how it looks like now in Discord? Then its easier to understand what you did.

![image](https://user-images.githubusercontent.com/58050604/144188486-72621204-8065-445f-a3b7-21db5c611156.png)

Reply:
Sounds good!